src/popup/popup.js

```diff
@@ -137,6 +155,15 @@ var Popup = Popup || {};
         chrome.tabs.sendMessage(govukTab.id, { trigger: 'toggleMetaTags' });
       });
     });
+
+    $('#toggle-design-mode').on('click', function(e) {
+      e.preventDefault();
+
+      chrome.tabs.query({ active: true, currentWindow: true }, function(tabs) {
+        var govukTab = tabs[0];
+        chrome.tabs.sendMessage(govukTab.id, { trigger: 'toggleDesignMode' });
+      });
+    });
```

Comment:
Can probably refactor out the duplication here, as we now have 3 functions doing exactly the same thing, just different selectors and triggers.